dashboard/app/models/reference_guide.rb

```diff
@@ -0,0 +1,75 @@
+require 'fileutils'
+
+# == Schema Information
+#
+# Table name: reference_guides
+#
+#  id                :bigint           not null, primary key
+#  display_name      :string(255)
+#  key               :string(255)      not null
+#  course_version_id :integer          not null
+#  content           :text(65535)
+#  position          :integer          not null
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#
+class ReferenceGuide < ApplicationRecord
+  belongs_to :course_version
+
+  validates_uniqueness_of :key, scope: :course_version_id
+
+  def serialize
```

Comment:
I think it would be great to follow the pattern Bethany started with programming_environments which allows us to not have to explicitly update this list every time we add a new property: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/programming_environment.rb#L51-L53